Src/Support/Google.Apis.Auth.AspNetCore/GoogleOpenIdConnectExtensions.cs

```diff
@@ -87,16 +88,20 @@ public static class GoogleOpenIdConnectExtensions
             {
                 // Google's OpenID authority URL.
                 options.Authority = "https://accounts.google.com";
-                // Response-code to get an id-token with profile information,
-                // and a code to use to get an access-token and refresh-token.
-                options.ResponseType = "id_token code";
+                // Response-type of "code" gets just an authorization-code from the authorization endpoint,
+                // then both an access-token and an id-token (JWT) from the token endpoint.
+                // For the id-token to be returned requires the "openid" scope to be set.
```

Comment:
Do you mean "for the id-token to be returned from the authorization endpoint"? (It's a bit unclear here.)

Reply:
Clarification added.